format/format.go

```diff
@@ -2,32 +2,41 @@ package format
 
 import (
 	"fmt"
-	"math"
-)
-
-const (
-	Thousand = 1000
-	Million  = Thousand * 1000
-	Billion  = Million * 1000
 )
 
 func HumanNumber(b uint64) string {
```

Comment:
We use this when setting the config layer for models which might be a bit problematic, let's update this a little and have two functions for now

`Parameters()` (used for `ollama show` etc)

`RoundedParameter()` (used in `images.go`)